app/poros/github/follower.rb

```diff
@@ -0,0 +1,8 @@
+class Follower
```

Comment:
I don't know if we want a Follower class necessarily, or an overarching Users class.. Then we'd filter down the users by whether or not they're following another specific user..